src/angular/projects/spark-angular/package.json

```diff
@@ -29,6 +29,7 @@
     "zone.js": "^0.8.26"
   },
   "dependencies": {
-    "lory.js": "^2.5.3"
+    "lory.js": "^2.5.3",
+    "lodash": "^4.17.15"
```

Comment:
Did you mean to leave lodash as a devDependency and also add it as a dependency? Do we need both?

Reply:
lodash was added as a dependency and removed as a peerDependency. It should only appear once in this file.

Reply:
Does anything need to be changed here or can we resolve the review?

Reply:
@bobwise, looks like you are changing the angular and spark versions to an old version here. Can you remove those changes? Them being '>=7.0.0 <=8.0.0' has been changed and the spark version has been updated too.